sce_presubmit_handler.py

```diff
@@ -0,0 +1,43 @@
+from subprocess import check_output, PIPE
+
+class ScePresubmitHandler:
+    test_commands = {
+        'Core-v4': ['npm run api-test --prefix ./Core-v4', 'npm run lint --prefix ./Core-v4', \
+            'npm run frontend-test --prefix ./Core-v4', 'npm run build --prefix ./Core-v4'],
+        'sce-rpc': ['npm run test --prefix ./sce-rpc', 'npm run lint --prefix ./sce-rpc', \
+            'pylint * --prefix ./sce-rpc'],
+        'SCE-discord-bot': ['npm run lint --prefix ./SCE-discord-bot'],
+        'dev': ['pylint * --ignore=Core-v4,sce-rpc,SCE-discord-bot']
+    }
+
+    def __init__(self, project):
+        self.project = project
+        """
+        this is a constructor, send in args.project from sce.py here.
+        """
+
+    @staticmethod
+    def run_test(test_name, command):
+        num_dots = 37 - len(test_name)
+        dots = "."
+        dots = dots * num_dots
+        dots_passed = dots + "passed"
+        dots_failed = dots + "failed"
+        try:
+            check_output(command, shell=True, stderr=PIPE)
```

Comment:
we should show errors. since youre leaving tomorrow, just open an issue on this repo to add failing test terminal output to this command